src/provider/cloud-element-templates/ElementTemplatesConditionChecker.js

```diff
@@ -60,6 +63,8 @@ export default class ElementTemplatesConditionChecker extends CommandInterceptor
     const propertiesToAdd = getMissingProperties(oldTemplate, newTemplate);
     const propertiesToRemove = getPropertiesToRemove(newTemplate, oldTemplate);
 
+    this._updateReferencedElement(element, oldTemplate, newTemplate);
+
```

Comment:
Do we see that this could this be used for other properties in the future? Just wondering about the generality of the function name

Reply:
I want it to be used for errors, signals, and escalations in the future.